microsoft-edge/devtools-guide-chromium/evaluate-performance/share-traces.md

```diff
@@ -0,0 +1,175 @@
+
+Using enhanced traces makes it possible to reliably resolve source code references found in imported traces to the actual runtime code in the **Sources** tool. Additionally, if source maps were presents when a trace was recorded, it will also be possible to resolve code references to their original source code.
+
+Enhanced traces can only be imported in Microsoft Edge, while normal traces are compatible with other browsers based on the Chromium engine.
+
+
+<!-- ====================================================================== -->
+## Set the default trace type
+
+By default, the **Performance** and **Memory** tools export data as normal traces, but it's possible to choose which type of traces are exported.
+
+To change the trace type:
+
+1. Open DevTools by pressing `Ctrl`+`Shift`+`I` (Windows, Linux) or `Command`+`Option`+`I` (macOS).
+
+1. In DevTools, in the upper right, click **Settings** (![Settings button](../media/settings-button-icon.msft.png)).
+
+1. Under **Persistence**, if you want to record enhanced traces, check **Export Enhanced Performance and Memory Traces**. Otherwise clear the option.
+
+   ![The Settings panel in DevTools, with the enhanced traces option checked](./share-traces-images/trace-type-setting.png)
+
+1. Enhanced traces can also optionally include console messages, script sources and DOM elements too. Choose what to include in enhanced traces by using these options:
+
+   * **Include console message in enhanced traces**.
+   * **Include script sources in enhanced traces**.
+   * **Include DOM snapshots in enhanced traces (experimental)**.
+
+   ![The Settings panel in DevTools, showing the other enhanced trace options](./share-traces-images/enhanced-trace-options.png)
+
+
+<!-- ====================================================================== -->
+## Export a trace from the Performance tool
+
+To export a performance recording from the **Performance** tool:
+
+1. Go to the website you want to record the performance of.
+
+1. Open the **Performance** tool.
+
+1. Press **Record**, execute the scenario you want to investigate the performance of by interacting with the webpage, and then press **Stop**.
+
+1. When the performance profile appears, click **Save profile** (![Save profile button](../media/export-icon.png)):
+
+   ![Microsoft Edge with DevTools, showing the Performance tool, with the Save profile button](./share-traces-images/exporting-perf-trace.png)
+
+1. Choose a location to save the trace file on your disk:
+
+   ![The Windows save dialog, showing the performance trace file being saved in a traces folder](./share-traces-images/saving-perf-trace.png)
+
+   The trace is a `.json` file if you set the default trace type to normal, and it is a `.devtools` file if you set the default trace type to enhanced.
```

Comment:
The alternative can be chosen by right-clicking the button, or by activating it with Ctrl+Space.

Reply:
Oh, that's covered below, so this can be a scratch.